src/java.base/share/classes/java/lang/Class.java

```diff
@@ -25,6 +25,9 @@
 
 package java.lang;
 
+import jdk.internal.javac.PreviewFeature;
+import jdk.internal.misc.PreviewFeatures;
```

Comment:
Can you move this so that it's with the other imports of jdk.internal classes?

Reply:
Changing in primary PR - https://github.com/openjdk/jdk/pull/13689